src/pages/index.js

```diff
@@ -97,6 +174,21 @@ const addFormValidator = new FormValidator(
 );
 addFormValidator.enableValidation();
 
+const editAvatarValidator = new FormValidator(
+  validationSettings,
+  avatarEditForm
+);
+editAvatarValidator.enableValidation();
+
+const deleteCardValidator = new FormValidator(
+  validationSettings,
+  deleteCardForm
```

Comment:
`deleteCardValidator` is not needed because it doesn't contain `inputs`, so no validation is needed